README.rst

```diff
@@ -172,46 +172,52 @@ let's use the UseDef chains combined with the ancestors.
 
 .. code:: python
 
-    >>> import gast as ast
-    >>> import beniget
-    >>> class CaptureX(ast.NodeVisitor):
-    ...
-    ...     def __init__(self, module_node, fun):
-    ...         self.fun = fun
-    ...         # initialize use-def chains
-    ...         du = beniget.DefUseChains()
-    ...         du.visit(module_node)
-    ...         self.chains = beniget.UseDefChains(du)
-    ...         self.ancestors = beniget.Ancestors()
-    ...         self.ancestors.visit(module_node)
-    ...         self.external = set()
-    ...
-    ...     def visit_Name(self, node):
-    ...         # register load of identifiers not locally definied
-    ...         if isinstance(node.ctx, ast.Load):
-    ...             def_ = self.chains.chains[node]
-    ...             try:
-    ...                 parents = self.ancestors.parents(def_.node)
-    ...             except KeyError:
-    ...                 return # a builtin
-    ...             if self.fun not in parents:
-    ...                     parent = self.ancestors.parentStmt(def_.node)
-    ...                     if parent not in self.external:
-    ...                         self.external.add(parent)
-    ...                         self.rec(parent)
-    ...
-    ...     def rec(self, node):
-    ...         "walk definitions to find their operands's def"
-    ...         if isinstance(node, ast.Assign):
-    ...             self.visit(node.value)
-    ...         # TODO: implement this for AugAssign etc
+    import gast as ast
```

Comment:
easier for copy past...

Reply:
the `>>>` are required for doctest compatibility :-) That way the examples in the README are part of the doc!